official/wide_deep/requirements.txt

```diff
@@ -0,0 +1 @@
+tensorflow==1.2.1
```

Comment:
Delete this file?

Reply:
This is so that users don't trip over API changes for new versions and anything in general due to versioning.